test/run-xspec-tests.sh

```diff
@@ -23,8 +23,13 @@
 #       LICENSE:  MIT License
 #
 #===============================================================================
-for xspectest in *.xspec; 
-do ../bin/xspec.sh $xspectest &> result.log; 
+for xspectest in *.xspec *.schut; 
+do 
+    if test "${xspectest#.schut}" = "$xspectest"; then
+        ../bin/xspec.sh -s $xspectest &> result.log;
+    else 
+        ../bin/xspec.sh $xspectest &> result.log;
+    fi
     if grep -q ".*failed:\s[1-9]" result.log || grep -q -E "\*+\sError\s(running|compiling)\sthe\stest\ssuite" result.log;
```

Comment:
This does not catch failures in the Schematron phase, such as "*** Error compiling the schematron on step 1".
(That is what's happening on Travis right now, I guess.)